stack/global/s3-private.tf

```diff
@@ -0,0 +1,20 @@
+module "s3-storage" {
```

Comment:
Where do we use this? Why do we need it? If we need it, can we name it by its purpose, e.g. `module "central-log-bucket"` or similar?